pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -168,6 +168,31 @@ func updateOrRemove(ctx *zedmanagerContext, status types.AppInstanceStatus) {
 	}
 }
 
+// PersistDirTotalAndDom0ReservedSizes
+// Return Values:
+//  total sizeBytes of /persist, sizeBytes reserved for Dom0, error
+func PersistDirTotalAndDom0ReservedSizes(
+	ctxPtr *zedmanagerContext) (uint64, uint64, error) {
+	deviceDiskUsage, err := disk.Usage(types.PersistDir)
+	if err != nil {
+		err := fmt.Errorf("Failed to get diskUsage for /persist. err: %s", err)
+		log.Errorf("diskSizeReservedForDom0: err:%s", err)
```

Comment:
I think this code belongs in the caller. Thus the function just calculates the diskReservedForDom0, which makes the choice of name more obvious.